styles/style.css

```diff
@@ -0,0 +1,149 @@
+@font-face {
+    font-family: 'Cake';
+    src: url(Cake_n_Truffles.ttf);
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    font-family: 'Cake', sans-serif;
+    color: #E4EBDB;
+}
+
+body {
+    padding: 1rem;
+}
+
+main {
+    background: linear-gradient(45deg, #008D36, #75AA31);
+    width: 1000px;
+    padding: 1rem;
+    margin: auto;
+}
+
+header {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    font-size: 2rem;
+    position: relative;
+    left: 30px;
+}
+
+header img {
+    height: 130px;
+}
+
+section {
+    display: flex;
+    justify-content: space-around;
+    margin-bottom: 5px;
+}
+
+#game {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 1rem;
+}
+
+#gameBoard {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(3, 1fr);
+    height: 35rem;
+    width: 35rem;
+    background-color: #94BF54;
+    background-image: url(../imageAssets/hashGrid.png);
+    background-position: center;
+    background-size: cover;
+    border-radius: 5%;
+}
+
+.tile {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.clickedX {
+    content: url(../imageAssets/smallX.png);
+    position: relative;
+    top: 15%;
+    left: 15%;
+}
+
+.clickedO {
+    content: url(../imageAssets/smallO.png);
+    position: relative;
+    top: 15%;
+    left: 15%;
+}
+
+#playerInfo {
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
I don't like how grid is being used here to stack items, but in the #game you're using flex to do exactly the same thing.

Suggest changing this to flex to be consistent (or visa versa)